src/web/app/components/question-types/question-edit-answer-form/msq-question-edit-answer-form.component.ts

```diff
@@ -0,0 +1,101 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  FeedbackMsqQuestionDetails,
+  FeedbackMsqResponseDetails,
+} from '../../../../types/api-output';
+import { DEFAULT_MSQ_QUESTION_DETAILS, DEFAULT_MSQ_RESPONSE_DETAILS } from '../../../../types/default-question-structs';
+import { QuestionEditAnswerFormComponent } from './question-edit-answer-form';
+
+/**
+ * The Msq question submission form for a recipient.
+ */
+@Component({
+  selector: 'tm-msq-question-edit-answer-form',
+  templateUrl: './msq-question-edit-answer-form.component.html',
+  styleUrls: ['./msq-question-edit-answer-form.component.scss'],
+})
+export class MsqQuestionEditAnswerFormComponent
+    extends QuestionEditAnswerFormComponent<FeedbackMsqQuestionDetails, FeedbackMsqResponseDetails> implements OnInit {
+
+  isMsqOptionSelected: boolean[] = Array(this.questionDetails.msqChoices.length).fill(false);
+  readonly NONE_OF_THE_ABOVE: string = 'None of the above';
```

Comment:
This seems like it's only used inside this class and not in the HTML template. Let's make this `private`, or even better, move this to outside the class definition (i.e. make it `const NONE_OF_THE_ABOVE = ...`).